src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Ball ball = new Ball();
+
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
100 is magic number
![image](https://user-images.githubusercontent.com/12864128/220328881-84ca8d79-14d7-49da-8fac-a2141e7d2650.png)